projects/igniteui-angular/src/lib/core/displayDensity.ts

```diff
@@ -16,6 +17,11 @@ export interface IDisplayDensity {
     displayDensity: DisplayDensity;
 }
 
+export interface IDensityChangedEventArgs {
+    oldDensity: DisplayDensity | string | IDisplayDensity;
```

Comment:
It's a bit odd that we emit `onDensityChanged` with different argument types. It should be emitted only with `DisplayDensity` in my opinion.
In addition we could rename `IDisplayDensity` to `IDisplayDensityOptions` to avoid confusion.